### PR TITLE
active is highlighted

### PR DESCRIPTION
**Pull request by jakobroehrl**
Signed-off-by: Jakob <jakob.roehrl@web.de>
questions:
- What is the right grey tone?
- If I open a card and reload the page the card is not highlighted after the reload. What could that be?

Thanks!

### CODE REVIEW

File: src/components/cards/CardItem.vue [hunk 1]
```diff
@@ -148,6 +151,10 @@ export default {
 		opacity: 0;
 	}
 
+	.current-card {
+		box-shadow: 0 0 3px 0 grey !important;
```
Comment (juliushaertl) [inline src/components/cards/CardItem.vue]: I'd say we use the same color but a bit bigger shadow:

```suggestion
		box-shadow: 0 0 5px 0 var(--color-box-shadow);
```

**Comment by juliushaertl** [comment]
> If I open a card and reload the page the card is not highlighted after the reload. What could that be?

When reloading the param seems to end up as a string while when navigating though clicking the parameter is an integer value

File: src/components/cards/CardItem.vue (hunk 1 above)
Comment (jakobroehrl) [reply src/components/cards/CardItem.vue]: I need the important Expression otherwise it won't work
Could we go up to 6px for better recognition?

**Comment by jakobroehrl** [comment]
> 
> 
> > If I open a card and reload the page the card is not highlighted after the reload. What could that be?
> 
> When reloading the param seems to end up as a string while when navigating though clicking the parameter is an integer value

@juliushaertl 
That's odd. I can retrace it. Should I change the comparison here to ==
https://github.com/nextcloud/deck/blob/d08bd23578ea4c05e27d3744094ee8f6e07ad356/src/components/cards/CardItem.vue#L103

File: src/components/cards/CardItem.vue (hunk 1 above)
Comment (juliushaertl) [reply src/components/cards/CardItem.vue]: 6px is also fine for me
try to move it into the actual .card section and use &.current-card to apply the class selector on top of that, so it gets applied after the regular card style in line 159

**Comment by jakobroehrl** [comment]
@juliushaertl 
This is still open "Should I change the comparison here to =="
The rest is working

**Pull request merged**